### ISSUE TITLE
Running an older Firefox instance

### ISSUE THREAD

**Issue by ashonline** [jlesage/docker-firefox#87]
Hi,

We have a requirement to be able to run Firefox 26 configured with the GWT plugin for the purpose of development, but due to the fact it's so old and basically completely unsecure, we're no longer allowed to have it installed on any development machines.

The reason for needing the old version of Firefox is because the GWT plugin was no longer supported after Firefox 26. The plugin is here: https://dl-ssl.google.com/gwt/plugins/firefox/gwt-dev-plugin.xpi

So we had an idea to run it in a docker container and lock it down, which led us to this rather wonderful docker image. We initially tried modifying the https://github.com/jlesage/docker-firefox/blob/master/Dockerfile and changing the value of FIREFOX_VERSION, but unsurprisingly this didn't work.

Would anyone have any pointers as to how we would accomplish this? 

Many thanks
Ash

**Comment by ashonline** [comment]
The alternative would be to start from scratch with a brand new Dockerfile that installs the zip file from here:

https://ftp.mozilla.org/pub/firefox/releases/26.0/linux-x86_64/en-GB/

But we're equally unsure how to proceed and make sure it gets properly integrated with docker-baseimage-gui.

**Comment by jlesage** [comment]
This container is based on Alpine Linux.  Unfortunately, the Alpine Linux software repository doesn't keep every versions of the (Firefox) packages.

I guess that you could try to use the Debian version of the baseimage.  You may then be able to run the version from the link you provided.

**Comment by ashonline** [comment]
That's pretty much what we ended up doing, creating a new image based on your rather excellent debian baseimage-gui image :-) then adding the firefox 26 zip file to it and then basically just building up the various dependencies.

One thing that we couldn't get to work though was to enable Firefox extensions on startup (in our case Google GWT), so it's necessary to do that manually through the Tools/Addons menu each time. We tried adding the following line to the Firefox prefs file, but unfortunately it seems to get wiped out by Firefox at some point during its startup sequence:

echo 'user_pref("extensions.enabledAddons", "gwt-dev-plugin%40google.com:1.26.2,%7B972ce4c6-7e08-4474-a285-3208198ce6fd%7D:26.0");' >> "/config/profile/prefs.js"

I copied that line from the one that Firefox itself adds when you go through the Tools/Addons menu.

If you know the answer then that would be great, but it's no great hardship for the team if not.
